src/main/java/firstSimpleJavaTasks/ArrayGenerating.java

```diff
@@ -0,0 +1,26 @@
+package firstSimpleJavaTasks;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ArrayGenerating {
```

Comment:
class names should be nouns, in this case, it would be better to call it ArrayGenerator

Reply:
classes renamed